pkg/core/chain/chain.go

```diff
@@ -359,11 +359,13 @@ func (c *Chain) AcceptBlock(ctx context.Context, blk block.Block) error {
 	c.p = &provisioners
 
 	if config.Get().API.Enabled {
-		store := capi.GetBuntStoreInstance()
-		err := store.StoreProvisioners(c.p, blk.Header.Height)
-		if err != nil {
-			log.Warn("Could not store provisioners on memoryDB")
-		}
+		go func() {
+			store := capi.GetBuntStoreInstance()
+			err := store.StoreProvisioners(c.p, blk.Header.Height)
+			if err != nil {
+				log.Warn("Could not store provisioners on memoryDB")
+			}
+		}()
```

Comment:
I assume the error is limited to the monitoring functions and does not influence the chain logic. If this i not the case (which I don't think), it may be better to use an `errgroup` or an error channel to report back the error, in case `store.StoreProvisioners` would fail

Reply:
Confirmed that the `store` is only for the monitoring APIs, therefore the error can be logged in a fire and forget manner